youtube_dl/extractor/nbc.py

```diff
@@ -109,18 +109,18 @@ class NBCSportsVPlayerIE(InfoExtractor):
     _VALID_URL = r'https?://vplayer\.nbcsports\.com/(?:[^/]+/)+(?P<id>[0-9a-zA-Z_]+)'
 
     _TESTS = [{
-        'url': 'https://vplayer.nbcsports.com/p/BxmELC/nbcsports_share/select/9CsDKds0kvHI',
+        'url': 'http://vplayer.nbcsports.com/p/BxmELC/chicago_embed/select/media/u4xMi3ngFKsS',
         'info_dict': {
-            'id': '9CsDKds0kvHI',
-            'ext': 'flv',
-            'description': 'md5:df390f70a9ba7c95ff1daace988f0d8d',
-            'title': 'Tyler Kalinoski hits buzzer-beater to lift Davidson',
-            'timestamp': 1426270238,
-            'upload_date': '20150313',
+            'id': 'u4xMi3ngFKsS',
+            'ext': 'mp4',
+            'description': 'md5:8fb3edf07c77cc5901c5f37fa019f87c',
+            'title': 'Cubs PGL: Who needs to step up in Contreras\' absence',
+            'timestamp': 1502319595,
+            'upload_date': '20170809',
             'uploader': 'NBCU-SPORTS',
         }
     }, {
-        'url': 'http://vplayer.nbcsports.com/p/BxmELC/nbc_embedshare/select/_hqLjQ95yx8Z',
+        'url': 'https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/media/qyc_v6mfDyWv',
```

Comment:
As well as this.

Reply:
https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/_hqLjQ95yx8Z

ERROR: This content expired on 9/17/14 12:23 PM.

Reply:
http://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/_hqLjQ95yx8Z

Reply:
It's a matching only test. Testing two identical URLs twice does not make any sense.

Reply:
Gotcha.  I will remove the duplicate test.

Reply:
You don't get it. **Don't touch tests**. After you change there are two tests with identical scheme. Testing this does not make any sense. Each tests must test **something special**.

Reply:
My apologies, a misunderstanding on my part.  With my latest commit, there is now only one test in place for this extractor - shall I revert the test suite back to the original state?  I believe those URLs were broken to begin with.

Reply:
Change 1 test to https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/9CsDKds0kvHI.
Change 2 test to https://vplayer.nbcsports.com/p/BxmELC/nbcsports_embed/select/media/_hqLjQ95yx8Z.

Reply:
Ah, now I see what you mean!  Thanks for the clarification.